group.py

```diff
@@ -2,4 +2,29 @@
 
 # Your code to go here...
 
-my_group =
+class Person:
+
+    def __init__(self, name, age,occupation):
+        self.name = name
+        self.age = age
+        self.occupation = occupation
+        self.relationships = {}
+
+    def add_relationship(self, connectedname, relationship):
+        self.relationships[connectedname]= relationship 
```

Comment:
Nice and concise way that allows you to scale this to as as many people as you want